lib/src/components/NxTab/NxTab.scss

```diff
@@ -6,16 +6,51 @@
  */
  @import '../../scss-shared/nx-colors';
  @import '../../scss-shared/nx-variables';
+ @import '../../scss-shared/nx-text-helpers';
+
+.nx-tabs {
+  margin: $nx-spacing-md 0 $nx-spacing-l 0;
+}
 
 .nx-tab {
-  border-bottom: 4px solid transparent;
+  border-bottom: 2px solid transparent;
   cursor: pointer;
   display: inline-block;
   list-style: none;
-  padding: $nx-spacing-xs $nx-spacing-l;
+  margin: 0 $nx-spacing-md -1px 0;
+  // subtract border thickness for consistent spacing
+  padding: $nx-spacing-md $nx-spacing-l ($nx-spacing-md - 2px) $nx-spacing-l;
+  text-align: center;
+  min-width: 76px;
 
   &.active {
-    border-bottom-color: $nx-background-color-interactive;
+    @include semi-bold();
+
+    border-bottom-color: #045dd1;
+    color: #045dd1;
+  }
+
+  &:hover {
+    @include semi-bold();
+  }
+
+  &:focus {
+    border-bottom-width: 1px;
+    box-shadow: 0 0 3px 1px transparentize(#02a6d9, 0.7);
+    outline: solid 1px #02a6d9;
```

Comment:
I would think that undoing the -1px bottom margin would be necessary here.  But for reasons I can't understand, it seems to make no difference.  I think it would be safer to add that change though.

Reply:
> I would think that undoing the -1px bottom margin would be necessary here. But for reasons I can't understand, it seems to make no difference. I think it would be safer to add that change though.

Yeah that's really weird, I reversed it here: https://github.com/sonatype/sonatype-react-shared-components/pull/153/commits/fd3679f6304ea30dc3ba1bbf58d806e063e779ab